contracts/multisig-ism/src/interface.sw

```diff
@@ -0,0 +1,34 @@
+library interface;
+
+use std::vm::evm::evm_address::EvmAddress;
+
+use hyperlane_message::Message;
+
+use multisig_ism_metadata::MultisigMetadata;
+
+abi MultisigIsm {
```

Comment:
Actually can you add a `validators_and_threshold` function? To adhere to https://github.com/hyperlane-xyz/hips/blob/main/HIP-3.md and so that the relayer just needs 1 query instead of 2

And we should also add a `module_type` function, a la https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/solidity/contracts/interfaces/IInterchainSecurityModule.sol#L5-L18

What was the difference between LEGACY_MULTISIG and MULTISIG again? I totally forgot about this stuff

Reply:
I guess LEGACY_MULTISIG describes this implementation and MULTISIG describes the MetaProxy-style approach